src/renderer/windows/settings/Settings.vue

```diff
@@ -396,10 +407,23 @@ window.ytmd.handleUpdateDownloaded(() => {
               <p class="description">
                 User is Authenticated:
                 <span v-if="lastFMSessionKey" style="color: #4caf50">Yes</span>
+                <button v-if="lastFMSessionKey" class="lastfm-logout-button" @click="logoutLastFM">Logout</button>
                 <span v-else style="color: #ff1100">No</span>
               </p>
             </div>
           </div>
+          <div v-if="lastFMEnabled" class="setting indented">
+            <YTMDSetting
```

Comment:
Is it possible to get the range to be right aligned
![image](https://github.com/ytmdesktop/ytmdesktop/assets/17199186/33ae8b55-0133-4775-80b8-b495f0ad26ea)

Similar to the zoom. I suspect that this is a problem with the range element and the intended class currently.
![image](https://github.com/ytmdesktop/ytmdesktop/assets/17199186/b7eb349d-4bff-4a54-8b57-3bba2759538d)

Reply:
I couldn't figure out how to get devtools on this settings windows look debug any of this. Any ideas?